### ISSUE TITLE
Tailwind config note cut short

### ISSUE THREAD

**Issue by pantheredeye** [redwoodjs/rw-shad#12]
It seems that the tailwind config note is being cut short with `...`

```
✔ Check for Tailwind setup...
✔ Installing packages...
✔ Update tailwind config...
  › Your Tailwind config already had customizations. We've done our best to merge the rw-shad settings with yours
  If things don't work as expected we recommend you rerun the setup with…
✔ Update index.css...
✔ Adding cn util...
✔ Adding components config...

```

**Comment by pantheredeye** [comment]
I expect that sentence has more instructions!

**Comment by Tobbe** [comment]
Thanks for reporting @pantheredeye 

![image](https://github.com/user-attachments/assets/f048da92-0784-447c-b40d-42a8ae4a3e4c)

Fix incoming! 🙂

**Comment by Tobbe** [comment]
Fixed in https://github.com/redwoodjs/setup-rw-shad/commit/6326a7fc24cb9ee957a3031eb608d4b5793335c2 and released as v1.0.1
